troupon/static/scss/partials/_components.scss

```diff
@@ -420,4 +424,75 @@ Other components.
       color: color(accent);
     }
   }
+}
+
+.jumbotron {
+  & > * {
+    margin-bottom: 15px;
+  }
+  .title {
+    color: color(h1);
+    font-size: 70px;
+    letter-spacing: -2px;
+  }
+}
+
+.breadcrumbs {
+  @include unstyled-h-list-mixin();
+  right: 10px;
+  
+  li.active {
+    color: color(accent);
+  }
+  li.delim {
+    font-size: font-size(breadcrumbs-caret);
+    color: color(icon-light);
+    vertical-align: middle;
+  }
+}
+
+.sidebar {
+  @include unstyled-list-mixin();
+  border-right: 1px solid color(divider);
+  li {
+    position: relative;
+    min-height: component-height(sidebar-item);
+    margin-right: -1px;
+    a, span {

```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3<br>Each selector in a comma sequence should be on its own single line